osu.Game.Rulesets.Osu/UI/OsuDrawableTouchInputHandler.cs

```diff
@@ -51,6 +57,10 @@ public class OsuDrawableTouchInputHandler : Drawable
             MaxValue = 10
         };
 
+        private Vector2 previousMousePosition;
+
+        private bool firstMouseDownApplied;
```

Comment:
Make `previousMousePosition` nullable for indication of first application, rather than defining a second member for it.

Reply:
Fixed on latest commit.